### PR TITLE
Modified safety ranges for the batteries according to the information on wiki

### PR DESCRIPTION
**Pull request by Davidhqm**
## Changes
- change 1: LP safety ranges for temperature, voltage, and current 
- change 2: HP safety ranges for temperature, voltage, and current

### CODE REVIEW

**Review by MiltFra — CHANGES_REQUESTED** [review]

File: src/sensors/bms_manager.cpp [hunk 1]
```diff
@@ -118,20 +118,20 @@ bool BmsManager::batteriesInRange()
...
+    if (battery.current < 0 || battery.current > 150 ) {  // current in 0A to 15A
       if (batteries_.module_status != previous_status_)
         log_.ERR("BMS-MANAGER", "BMS LP %d current out of range: %d", i, battery.current);
       return false;
     }
 
-    if (battery.average_temperature < 10
-        || battery.average_temperature > 60) {  // temperature in 10C to 60C
+    if (battery.average_temperature < 0
+        || battery.average_temperature > 80) {  // temperature in 0C to 80C
```
[the remark this answers is not in the document]
Comment (kshxtij) [reply src/sensors/bms_manager.cpp:134]: They said 80 degrees is the "shut-off" value where we should stop the pod instantly. They asked if we could implement a warning of some sort at 70 degrees, which i added to the to-do list.

[the remark this answers is not in the document]
Comment (MiltFra) [reply src/sensors/bms_manager.cpp:134]: Do it here and add a comment explaining the above. No need to delay. Same goes for all the other values. Make sure we don't lose how we got that information in the first place! (I.e. document the hard limits)

**Review by kshxtij — APPROVED** [review]

File: src/sensors/bms_manager.cpp [hunk 2]
```diff
@@ -131,7 +131,7 @@ bool BmsManager::batteriesInRange()
     }
 
     if (battery.average_temperature < 0
-        || battery.average_temperature > 80) {  // temperature in 0C to 80C
+        || battery.average_temperature > 70) {  // temperature in 0C to 70C (70C is the upper safe limit; 80 is the shutdown temperature)
```
Comment (MiltFra) [inline src/sensors/bms_manager.cpp:134]: please put the comment above the if to not exceed the line limit. auto format after; it should be nicer then

**Review by MiltFra — APPROVED** [review]
I fixed the formatting. The changes look fine to me now.

**Pull request merged**